data/final/currencies.json

```diff
@@ -1211,7 +1210,7 @@
     "iso_4217_3": "USD",
     "number_decimals": 2,
     "symbols": {
-      "primary": "US$",
```

Comment:
@mbryzek this was changed automatically (underlying data `cldr-currencies`) has the primary symbol here as `$`

Do we need to override this? Want to be careful as I know we've had issues in the past with showing the correct symbol.

Reply:
yeah - we need to keep `US$` - thanks for highlighting